contact.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Pet Store</title>
+  <link rel="icon" href="img\petstore_logo.png" sizes="32x32">
+	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato|Satisfy">
+	<link rel="stylesheet" href="css/bootstrap.min.css">
+	<link rel="stylesheet" href="css/styles.css">
+	<link rel="stylesheet" href="css\font-awesome.min.css">
+</head>
+<body>
+	<div class="container">
+		<div class="row">
+			<div class="col-md-12">
+				<a href="index.html"></a>
+					<img src="img\petstore_logo.png" alt="petstore_logo" class="logo">
+					<h3 class="logo-text">Pet Store</h3>
+				</a>
+				<ul class="nav nav-pills pull-right">
+				  <li role="presentation"><a href="index.html">Home</a></li>
+				  <li role="presentation"><a href="#">Products</a></li>
+				  <li role="presentation" class="active"><a href="#">Contact Us</a></li>
+				</ul>
+			</div>
+		</div>
+	</div>
+
+	<!-- end of nav -->
+  <div class="container">
+    <div class="row">
+      <div class="col-md-7">
+        <!-- veterinarians -->
+        <div class="row">
+        <!-- joe -->
+          <div class="col-md-4">
+            <img src="img\doctor1.jpg" alt="Joe Smith" class="img-responsive img-rounded shadow">
```

Comment:
does this img work? it's typically written as `img/doctor1.jpg`

Reply:
Yes, I copied the file path. That is how it pasted to my code.